install/css/install.css

```diff
@@ -191,6 +191,10 @@ input[type="url"] {
 	width: 100%;
 }
 
+input[type="number"] {
```

Comment:
I don't understand. If the style is changed for .elgg-input-number only you might end with differently styled number input fields if someone adds a input field not using the elgg-input-number class.